test/py/test_views_index.py

```diff
@@ -118,6 +118,7 @@ def test_get_config(mock_request):
         index = Index(mock_request)
         assert index.get_config() == {
             'application_url': 'http://example.com/',
```

Comment:
is this one still used?

Reply:
Yes, the application URL is still needed for the search service.